numpy/core/src/multiarray/datetime.c

```diff
@@ -778,8 +778,9 @@ parse_datetime_extended_unit_from_string(char *str, Py_ssize_t len,
         goto bad_input;
     }
     out_meta->base = parse_datetime_unit_from_string(substr,
-                                        substrend-substr, metastr);
-    if (out_meta->base == -1) {
+                                                     substrend - substr,
+                                                     metastr);
```

Comment:
nit: Not a huge fan of this type of linewrapping, as it always gets messed up when functions are renamed.

Reply:
I'm not a big fan either, but it is PEP7 and pretty common in the NumPy C code. The NumPy C style guide is more forgiving
```
* Breaking long lines: if you can, break after commas in the
  outermost argument list.  Always indent continuation lines
  appropriately, e.g., ::

        PyErr_SetString(PyExc_TypeError,
                "Oh dear, you messed up.");

  Here appropriately means at least two tabs. It isn't necessary to
  line everything up with the opening parenthesis of the function
  call.
```

Reply:
I think I prefer the two tabs approach, as

    PyErr_SetString(
            PyExc_TypeError,
            "Oh dear, you messed up.");

Although for calls to python builtins which won't change, it doesn't matter. Our internal functions do seem to change name often, especially when we need an extended API which is wrapped by the external C api.